src/storage/test/DeleteVertexTest.cpp

```diff
@@ -20,9 +20,10 @@ namespace storage {
 TEST(DeleteVertexTest, SimpleTest) {
     fs::TempDir rootPath("/tmp/DeleteVertexTest.XXXXXX");
     std::unique_ptr<kvstore::KVStore> kv(TestUtils::initKV(rootPath.path()));
+    auto schemaMan = TestUtils::mockSchemaMan();
```

Comment:
Do we need `schema manager` at here ?

Reply:
Yes, we need check the existing index via schemaMan.